kebechet/cli.py

```diff
@@ -41,27 +41,55 @@ def _print_version(ctx, _, value):
 
 @click.group()
 @click.pass_context
-@click.option('-v', '--verbose', is_flag=True, envvar='KEBECHET_VERBOSE',
-              help="Be verbose about what's going on.")
-@click.option('--version', is_flag=True, is_eager=True, callback=_print_version, expose_value=False,
-              help="Print version and exit.")
+@click.option(
+    "-v",
+    "--verbose",
+    is_flag=True,
+    envvar="KEBECHET_VERBOSE",
+    help="Be verbose about what's going on.",
+)
+@click.option(
+    "--version",
+    is_flag=True,
+    is_eager=True,
+    callback=_print_version,
+    expose_value=False,
+    help="Print version and exit.",
+)
 def cli(ctx=None, verbose=0):
-    """The CLI."""
+    """CLI."""
     if ctx:
-        ctx.auto_envvar_prefix = 'KEBECHET'
+        ctx.auto_envvar_prefix = "KEBECHET"
 
     if verbose:
         _LOGGER.setLevel(logging.DEBUG)
         _LOGGER.debug("Debug mode turned on")
         _LOGGER.debug("Kebechet version: %r", kebechet_version)
 
 
-@cli.command('run')
-@click.argument('configuration', metavar='config', envvar='KEBECHET_CONFIGURATION_PATH')
+@cli.command("run")
+@click.argument("configuration", metavar="config", envvar="KEBECHET_CONFIGURATION_PATH")
 def cli_run(configuration):
     """Run Kebechet using provided YAML configuration file."""
     config.run(configuration)
 
 
-if __name__ == '__main__':
+@cli.command('run-results')
+@click.argument('origin', metavar='org', envvar='GITORIGIN')
+@click.argument('service', envvar='KEBECHET_SERVICE')
+@click.argument('analysis_id', metavar='id', envvar='ANALYSIS_ID')
```

Comment:
It would be better to turn these into options instead of positional arguments and you don't need to specify them in `app.sh` script (as they will use `envvar`). Also please prefix `KEBECHET_` everything which is Kebechet specific.